pallets/msa/src/lib.rs

```diff
@@ -179,11 +161,19 @@ pub mod pallet {
 		#[pallet::constant]
 		type MaxSignaturesStored: Get<Option<u32>>;
 
-		/// The origin that is allowed to create providers
+		/// The origin that is allowed to create providers.  It is blocked by a call filter for mainnet.
 		type CreateProviderOrigin: EnsureOrigin<Self::RuntimeOrigin, Success = Self::AccountId>;
```

Comment:
We can remove `CreateProviderOrigin` and just stick with the `let sender = ensure_signed(origin)?;` in the function since that is all this is doing now.